.github/workflows/npm-publish.yml

```diff
@@ -1,29 +1,35 @@
-# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
-# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
-
 name: Publish to Npm
 
-on: push
+on:
+  workflow_run:
+    workflows: ["ESLint", "Typecheck", "Test"]
+    types:
+      - completed
+  push:
+    branches:
+      - master
```

Comment:
I think it would be better if we also (or only) publish it npm when we merge something to master. I don't want to push something to master directly

Reply:
That should be done with Repo Settings. I don't have a good solution for pull_requests merged. 
The closest I could find was:
https://github.community/t/trigger-workflow-only-on-pull-request-merge/17359/12